.gitlab/scripts/download_tracer_artifacts.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -e
+
+GITLAB_TOKEN=$(aws ssm get-parameter \
+    --region us-east-1 \
+    --name "ci.$CI_PROJECT_NAME.serverless-gitlab-token" \
+    --with-decryption \
+    --query "Parameter.Value" \
+    --out text)
+
+TRACER_PROJECT_ID=348
+
+# Clean environment variables if they are set as 'placeholder'
```

Comment:
it would be great to wrap some of this code in functions for maintainability!

Reply:
not doing it tbh, this is very specialized for .NET